sep490-frontend/src/app/modules/enterprise/components/enterprise-invitation/enterprise-invitation.component.ts

```
@@ -0,0 +1,10 @@
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'app-enterprise-invitation',
+  templateUrl: './enterprise-invitation.component.html',
+  styleUrl: './enterprise-invitation.component.css'
+})
+export class EnterpriseInvitationComponent {
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)